src/react-native/18-maps/maps.md

```diff
@@ -107,11 +149,42 @@ If you’ve implemented the solution correctly, the Map should be rendering in y
 
 ## Objective 2: Add restaurant pins with tooltips to map
 
-TODO
```

Comment:
This text should explain what we’re gonna do, e.g. “now that we have a map, let’s add markers for each one of the restaurants. When we tap on them, we will navigate to the restaurant detail page, just like we do in the list view.”